controllers/seedimage_controller.go

```diff
@@ -539,6 +508,57 @@ func fillBuildImagePod(name, namespace, buildImg, baseImg, configMap string, pul
 	return pod
 }
 
+func defineInitContainers(baseImg, buildImg string, pullPolicy corev1.PullPolicy) []corev1.Container {
+	const baseIsoPath = "/iso/base.iso"
+
+	containers := []corev1.Container{}
+	buildCommands := []string{
+		fmt.Sprintf("xorriso -indev %s -outdev /iso/elemental.iso -map /overlay / -boot_image any replay", baseIsoPath),
+	}
+
+	if util.IsHTTP(baseImg) {
+		buildCommands = append([]string{fmt.Sprintf("curl -Lo %s %s", baseIsoPath, baseImg)}, buildCommands...)
+	} else {
+		// If baseImg is not an HTTP url assume it is an image reference
+		containers = append(
+			containers, corev1.Container{
+				Name:            "baseiso",
+				Image:           baseImg,
+				ImagePullPolicy: corev1.PullIfNotPresent,
+				Command:         []string{"busybox", "sh", "-c"},
+				Args:            []string{fmt.Sprintf("busybox cp /elemental-iso/*.iso %s", baseIsoPath)},
```

Comment:
The base iso is copied to the shared volume.

Reply:
pretty neat :-)